website/assets/styles/pages/get-started.less

```diff
@@ -9,7 +10,7 @@
     color: #6a67fe;
   }
   #primary-button {
-      color: #fff;
+    color: #fff;
   }
```

Comment:
is this a tab?  I pasted it in the javascript console and it acted like some sort of special character  (Looks to me the change on this line was accidental, so I think we should revert it)

Reply:
Actually, looking at it w/ fresh eyes it looks like maybe that was a github diff issue